app/src/main/java/com/goayo/debtify/view/adapter/GroupViewAdapter.java

```diff
@@ -148,11 +159,30 @@ public GroupViewHolder(@NonNull View itemView) {
         @SuppressLint("SetTextI18n")
         public void setGroupData(IGroupData group, String currentLoggedInUsersPhoneNumber) {
             groupName.setText(group.getGroupName());
-            //TODO ("SET BALANCE IN GROUP_CARDVIEW")
+            StringBuilder sb = new StringBuilder();
+
+            for (IUserData iu : group.getIUserDataSet()){
+                sb.append(iu.getName());
+                sb.append(", ");
+            }
+
+            groupMembers.setText(sb.substring(0, sb.length()-2)); //Make substring of whole string except the last added ", ".
             try {
+                int i = group.getUserTotal(currentLoggedInUsersPhoneNumber).compareTo(new BigDecimal(0));
```

Comment:
defuk is this c style return value
![WutFace](https://user-images.githubusercontent.com/54945977/95753269-aad0a300-0ca1-11eb-8b0f-113e65a589f7.png)